addons/monitoring/main.tf

```diff
@@ -21,7 +21,6 @@ module "grafana" {
   k8s_cluster_ca_certificate = data.terraform_remote_state.infra.outputs.cluster_ca_certificate
   k8s_client_token           = data.terraform_remote_state.infra.outputs.token
   namespace                  = "tsb-monitoring"
-  admin_password             = var.tsb_password
```

Comment:
hopefully this is not the issue to use `var.tsb_password` it will just keep the UX more streamlined.